### DISCUSSION TITLE
Update API Docs on the UI with all public endpoints

### DISCUSSION THREAD

**Discussion by ghost** [JiscSD/octopus#40] [Ideas]
Possibly linked from -> https://github.com/JiscSD/octopus/issues/30

The docs on the /documentation route of the UI need updating to include all public endpoints. 
Components have been setup for this, the data just needs adding.

Additional style updates will be needed depending on the content.

**Comment by nasainsbury** [comment]
Good discussion point.

Just to add (I know @AshleyRedmanJisc knows this), but for the benefit of anyone else.

**ALL** endpoints are technically public, but there are endpoints that we'd recommend people not using, such as creating a publication. The reason behind this is that creating a publication requires many fields to be filled in, along with approvals, and a UI generally helps with this. That being said, it's perfectly possible to create and publish a publication via the API.

As part of the discussion it's worth thinking about API support, and what endpoints we want to support active use of, and what we want to deter (recommend UI only).
